corehq/apps/saved_reports/models.py

```diff
@@ -716,3 +716,22 @@ def get_latest(cls):
             return cls.objects.order_by('-end_datetime')[0]
         except IndexError:
             return None
+
+
+class ScheduledReportRecord(models.Model):
+    class States(object):
+        queued = 0
+        succeeded = 1
+        failed = 2
+        skipped = 3
+        choices = [
+            (queued, 'Queued'),
+            (succeeded, 'Succeeded'),
+            (failed, 'Failed'),
+            (skipped, 'Skipped'),
+        ]
+
```

Comment:
could be useful to have `domain` in here

Reply:
Yeah that makes sense. It's technically "duplicate" information because it's already contained in the `ReportNotification`, but that's in couch so there's no way to query on it with a join. I'll add a column here.

Reply:
Hmm, on second thought, domain isn't actually available at the time we create a `ScheduledReportRecord`. I think I'm going to skip this for now, and rely on looking up by `ReportNotification` id if we're trying to debug something specific.